reference/forms/types/collection.rst

```diff
@@ -330,6 +331,14 @@ rendering your form, having the entire "form row" may be easier for you.
 For details on how to actually use this option, see the above example as well
 as :ref:`cookbook-form-collections-new-prototype`.
 
+prototype_data
+~~~~~~~~~~~~~~
+
+**type**: ``mixed`` **default**: ``null``
+
+Allows you to specify data for the prototype. This is useful if you want to
+have the fields filled with some basic values.
```

Comment:
Yeah, I like `some initial values` better too. To address xabbuh's concerns - what about something along the lines of `Each new row added will initially contain the data set by this option`?